lib/contracts/contributionreward.ts

```diff
@@ -118,6 +120,19 @@ export class ContributionRewardWrapper extends ExtendTruffleContract {
       throw new Error("beneficiary is not defined");
     }
 
+    const controller = await this.getController(options.avatar);
+    const schemeParams = await controller.getSchemeParameters(this.address, options.avatar);
+
+    const orgNativeTokenFee = (await this.contract.parameters(schemeParams))[0];
+    if (Config.get("autoApproveTokenTransfers") && (orgNativeTokenFee > 0)) {
+      /**
+       * approve immediate transfer of native tokens from msg.sender to the avatar
+       */
+      const avatarService = new AvatarService(options.avatar);
+      const token = await avatarService.getNativeToken();
+      await token.approve(this.address, orgNativeTokenFee, { from: Utils.getDefaultAccount() });
```

Comment:
why did this work before, without approval?

Reply:
There were no tests for the case where orgNativeTokenFee was > 0.  Can't speak for Alchemy.